### PR TITLE
feat: add a tokenColor field for tokens

### PR DESCRIPTION
**Pull request by nahoc10**
Adds a `tokenColor` field to the `tokens`, useful to map to a specific color on the frontend. It doesn't return an average of the colors, but the most dominant color instead.

Also, noticed that these 2 images aren't proper images format:
- https://raw.githubusercontent.com/gondola-finance/frontend-v2/master/src/assets/icons/colour.png
- https://raw.githubusercontent.com/ava-labs/subnet-assets/main/chains/43114/0xC931f61B1534EB21D8c11B24f3f5Ab2471d4aB50/logo.png

Hence they should probably be removed or changed.

### CODE REVIEW

File: actions/createChainsAndERC20List.js [hunk 1]
```diff
@@ -3,17 +3,45 @@ import path from 'path';
...
+            return {
+              ...token,
+              tokenColor: color.hex,
+            };
+          } catch (err) {
+            console.log(err);
+          }
+        }
+        return token;
+      });
```
Comment (connorch) [inline actions/createChainsAndERC20List.js:31]: Nice!

Can we abstract this into a method in a separate `createToken` file (similar to how we have the `createChain` file) for the sake of consistency?

File: package.json [hunk 2]
```diff
@@ -8,13 +8,16 @@
...
-    "tsm": "^2.2.1"
+    "tsm": "2.2.1"
   },
   "dependencies": {
-    "@actions/core": "^1.8.2",
-    "ajv": "^8.11.0",
-    "ajv-formats": "^2.1.1"
+    "@actions/core": "1.8.2",
+    "ajv": "8.11.0",
+    "ajv-formats": "2.1.1"
```
Comment (nahoc10) [inline package.json:21]: pinned dependencies

**Comment by connorch** [comment]
@dovbrand @markgmilner Cohan wasn't able to get JS native `async` and `await` to work properly - any tips for getting that working?  And is it the node version that's causing the build to fail?

**Comment by markgmilner** [comment]
> @dovbrand @markgmilner Cohan wasn't able to get JS native `async` and `await` to work properly - any tips for getting that working? And is it the node version that's causing the build to fail?

I'll take a look.  Let me pull down the repo.

**Comment by markgmilner** [comment]
@connorch @nahoc10 the issue was the github workflow did not have a step to install deps, which wasn't a problem before since the lists were built using only native node modules.  I made a few other changes.  PTAL.

**Pull request merged**